api/package.json

```diff
@@ -42,6 +42,7 @@
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "generate-report": "nyc report --reporter=html",
     "lint": "eslint src --ext ts",
+    "lint-fix": "eslint src --ext ts --fix",
     "lint-modified": "bash -c '$(npm bin)/tslint -c tslint.json --project tsconfig.json | grep -f <(git ls-files --modified) --color=always'",
```

Comment:
Running `npm run lint --fix` removes the fix-flag. So I added this new command `npm run lint-fix`